ci/ansible/roles/pulp-test-populate/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Start Pulp
```

Comment:
Variables passed at the command line with `-e` always, always, always win. They're the highest priority type of variable, and will override whatever's in a playbook.